firebase/rules.bolt

```diff
@@ -426,11 +521,13 @@ type PublicLife extends LifeBase {
 /groups/{GroupId} is Group;
 /groups/GroupId/players/{PlayerId} is PrivatePlayer;
 /guns/{GunId} is Gun;
+/queuedNotification/{QueuedNotificationId} is QueuedNotification;
+/maps/{MapId} is MapType;
 /missions/{MissionId} is Mission;
 /playersPrivate/{PlayerId} is PrivatePlayer;
```

Comment:
in the firebase, this was renamed to privatePlayers and publicPlayers (to be consistently the same name as their type)